IndicoToolkit.Tests/ExtractionsTests.cs

```diff
@@ -67,5 +73,86 @@ public void TestRemoveByConfidence()
             }
         }
 
+        [Fact]
+        public void TestRemoveExceptMaxConfidence()
+        {
+            List<Prediction> predictions = new List<Prediction>();
+            JObject newConfidence = new JObject{
+                { "testLabel", .99f }
+            };
+            predictions.Add(Utils.CreatePrediction(text:"keep this", confidence: newConfidence));
+            predictions.Add(Utils.CreatePrediction(text:"remove this"));
+            Extractions extractionsObject = new Extractions(predictions);
+            extractionsObject.removeExceptMaxConfidence(new List<string>(){"testLabel"});
+            Assert.Equal(extractionsObject.Preds.Count, 1);
+            Assert.Equal(extractionsObject.RemovedPreds.Count, 1);
+            Assert.Equal((string)extractionsObject.Preds[0].getValue("text"), "keep this");
+            Assert.Equal((string)extractionsObject.RemovedPreds[0].getValue("text"), "remove this");
+        }
+
+        [Fact]
+        public void TestExistMultipleValsForLabel()
+        {
+            List<Prediction> predictions = new List<Prediction>();
+            predictions.Add(Utils.CreatePrediction(text:"A"));
+            predictions.Add(Utils.CreatePrediction(text:"B"));
+            predictions.Add(Utils.CreatePrediction(label: "otherLabel", text: "C"));
+            Extractions extractionsObject = new Extractions(predictions);
+            Assert.True(extractionsObject.existMultipleValsForLabel("testLabel"));
+            Assert.False(extractionsObject.existMultipleValsForLabel("otherLabel"));
+        }
+
+        [Fact]
+        public void TestGetMostCommonTextValue()
+        {
+            List<Prediction> predictions = new List<Prediction>();
+            predictions.Add(Utils.CreatePrediction(text:"this"));
+            predictions.Add(Utils.CreatePrediction(text:"this"));
+            predictions.Add(Utils.CreatePrediction(text:"not this"));
+            Extractions extractionsObject = new Extractions(predictions);
+            string mostCommonTextValue = extractionsObject.getMostCommonTextValue("testLabel");
+            Assert.Equal(mostCommonTextValue, "this");
+        }
+
+        [Fact]
+        public void TestToCSV()
```

Comment:
I have some questions about supporting this feature due to third party packages  ... let's discuss later today